modules/cf-core/src/protocol/propose.ts

```diff
@@ -44,7 +44,7 @@ export const PROPOSE_PROTOCOL: ProtocolExecutionFlow = {
 
     const preProtocolStateChannel = stateChannelsMap.get(multisigAddress)
       ? stateChannelsMap.get(multisigAddress)!
-      : StateChannel.createEmptyChannel(multisigAddress, [
+      : StateChannel.createEmptyChannel(multisigAddress, network.ProxyFactory, [
```

Comment:
note: this is done to allow for virtual proposals, but it still makes me a *bit* uneasy..